depthai_nodes/ml/parsers/dmcount.py

```diff
@@ -0,0 +1,49 @@
+import depthai as dai
+
+from ..messages.creators import create_density_message
+
+
+class DMCountParser(dai.node.ThreadedHostNode):
```

Comment:
This parser looks pretty general. What about renaming it to `DensityParser` or something similar?

Reply:
Yeah that makes sense! However, I'd also merge it together with `MonocularDepthParser` to make it even more general (both parsers output 2d array of float values). So I'd maybe leave it named as it is for now and fix both parsers in the next PR!